packages/editor-ui/src/stores/projects.store.ts

```diff
@@ -49,19 +49,19 @@ export const useProjectsStore = defineStore('projects', () => {
 	);
 	const teamProjects = computed(() => projects.value.filter((p) => p.type === ProjectTypes.Team));
 	const teamProjectsLimit = computed(() => settingsStore.settings.enterprise.projects.team.limit);
-	const teamProjectsAvailable = computed<boolean>(
+	const isTeamProjectsAvailable = computed<boolean>(
```

Comment:
👍 much better with the prefix. Tho to be grammatically correct, we should name this `areTeamProjectsAvailable`, as it's plural project**s**